### ISSUE TITLE
Broken @material-ui dependency? 

### ISSUE THREAD

**Issue by ArcticTee** [waoai/react-image-annotate#6]
Hi, I am trying to include your Component in a React project of mine.

Running "@material-ui/core": "^4.3.1",

Getting the following errors:
```
./node_modules/react-image-annotate/Theme/index.js
Module not found: Can't resolve '@material-ui/core/styles/MuiThemeProvider' in '.../node_modules/react-image-annotate/Theme'
```

When I go back to a Version 3.9.3 of @material-ui the MuiThemeProvider is found.
But the @material-ui/lab/Slider dependency from material-survey will be lost.

`Can't resolve '@material-ui/lab/Slider' in '.../node_modules/material-survey/components/RatingQuestion'`

Or it may be related to [this]( https://github.com/mui-org/material-ui/issues/17900#issuecomment-542590380)

Any suggestions?

**Comment by oliviertassinari** [comment]
Sorry for the trouble, does the import come from the auto-import feature of your IDE?
I would encourage the following:

```diff
-import { makeStyles } from "@material-ui/core/styles"
-import createMuiTheme from "@material-ui/core/styles/createMuiTheme"
-import MuiThemeProvider from "@material-ui/core/styles/MuiThemeProvider"
+import { makeStyles, createMuiTheme, ThemeProvider as MuiThemeProvider } from "@material-ui/core/styles"
```

**Comment by ArcticTee** [comment]
What do you mean by 

> does the import come from the auto-import feature of your IDE?

I used the npm package and tried to use your `Annotator` Component

`import {Annotator} from "react-image-annotate"`

I guess the fix for your Theme would work.

**Comment by oliviertassinari** [comment]
@ArcticTee Sorry, it was a question for the author of the package.

**Comment by ArcticTee** [comment]
Sorry, confused you with the author or contributors of this package😂

**Comment by oliviertassinari** [comment]
Your best workaround is to freeze @material-ui/core to v4.5.0.

**Comment by ArcticTee** [comment]
I solved it by using    
```
    "@material-ui/core": "4.4.0",
    "@material-ui/lab": "4.0.0-alpha.22",
```
in my `package.json  `

**Comment by seveibar** [comment]
@ArcticTee thanks for reporting, assigning myself to fix this issue.

**Comment by oliviertassinari** [comment]
Notice that Material-UI v4.5.2 will soon come out with a warning instead of an exception for the import of this private module.

**Comment by seveibar** [comment]
fixed in 0.1.28
